data.sql

```diff
@@ -1,17 +1,38 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animal (name) VALUES ('Luna');
+INSERT INTO animal (name) VALUES ('Daisy');
+INSERT INTO animal (name) VALUES ('Charlie');
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES (1, 'Agumon', 'Feb 03,2020', 0, true, 10.23);
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES (2, 'Gabumon', 'Nov 15, 2018', 2, true, 8);
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES (3, 'Pikachu', 'Jan 07, 2021', 1, false, 15.04);
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES (4, 'Devimon', 'May 12, 2017', 5, true, 11);
+
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (5, 'Charmander', 'Feb 03, 2020', 0, false, 11);
```

Comment:
- [ ] Good job on the queries but please kindly take note that the weight for this query is `-11kg`. Kindly ensure your queries stick to the project guidelines.